launcher/src/backend/ethereum-services/GethService.js

```diff
@@ -9,6 +9,7 @@ export class GethService extends NodeService {
 
         const service = new GethService();
         service.init(null,
+            "GethService",
```

Comment:
shouldn't there be another `null` as argument for the id?

Reply:
or switch `null` with `"GethService"` one line up?

Reply:
ohhh man and i even tripple checked 😅